content/blog/2020-08-19-train-a-sequence-model-with-poutyne/utils.py

```diff
@@ -0,0 +1,39 @@
+import gzip
```

Comment:
Not sure I follow why is there a .py file associated with the blog ? Is it an error or is it used some place I missed ?

Reply:
Yeah error of a previous approach. will delete.